src/components/ConversationSettings/BanSettings/BannedItem.vue

```diff
@@ -54,12 +54,12 @@ export default {
 	computed: {
 		banInfo() {
 			return [
-				t('spreed', '<strong>Banned by:</strong> {actor}', { actor: this.ban.actorId },
+				t('spreed', '<strong>Banned by:</strong> {actor}', { actor: this.ban.moderatorDisplayName },
 					undefined, { escape: false, sanitize: false }),
```

Comment:
Still XSS via user display name.
Please just put the html + label into the source of the frontend and add the input manually